homeassistant/components/forecast_solar/strings.json

```diff
@@ -24,7 +24,8 @@
           "damping": "Damping factor: adjusts the results in the morning and evening",
           "inverter_size": "Inverter size (Watt)",
           "declination": "Declination (0 = Horizontal, 90 = Vertical)",
-          "modules power": "Total Watt peak power of your solar modules"
+          "modules power": "Total Watt peak power of your solar modules",
+          "horizon": "Horizon heights as comma seperated list of degrees (0,0,0,10,20,20,20)"
```

Comment:
- Why would it need multiple horizon heights?
- We should not accept or ask users for comma-separated input.

Reply:
I mainly copied this option 1 to 1 from the forecast.solar documentation. I also prefer to see the input other than a comma separated list.

Reply:
wow, this is one hell of an advanced option. I'm honestly not even sure if we should add it (and if we do, it definitely needs to be shown only to users that have enabled advanced mode in Home Assistant).

My guess would be, that this feature would be rarely used.

Reply:
We can also leave it for now and first get started with migrating the damping to a damping morning / evening?

Reply:
sounds good